packages/components/link/src/link.story.js

```diff
@@ -16,13 +23,39 @@ storiesOf('Components|Links', module)
   .add('Link', () => (
     <Router>
       <Section>
-        <Link
-          tone={select('Tone', ['primary', 'inverted', 'secondary'])}
-          to={text('to', '/foo/bar')}
-          isExternal={boolean('isExternal', false)}
-        >
-          {text('label', 'Accessibility text')}
-        </Link>
+        <Spacings.Stack>
+          <Link
+            tone={select('Tone', ['primary', 'inverted', 'secondary'])}
+            to={text('to', '/foo/bar')}
+            isExternal={boolean('isExternal', false)}
+          >
+            {text('label', 'Accessibility text')}
+          </Link>
+          <Text.Body>
+            <Link tone="primary" to="/foo/bar" isExternal>
+              text
+            </Link>
+          </Text.Body>
+          <Card>
+            <Link tone="primary" to="/foo/bar" isExternal>
+              text
+            </Link>
+          </Card>
+          <ContentNotification type="error">
+            <Link tone="primary" to="/foo/bar" isExternal>
+              text
+            </Link>
+          </ContentNotification>
+          <CheckboxInput
+            value="foo-radio-value"
+            onChange={(event) => alert(event.target.value)}
+            isChecked={true}
+          >
+            <Link tone="primary" to="/foo/bar" isExternal>
+              text
+            </Link>
+          </CheckboxInput>
```

Comment:
This is just a temporary change to showcase `<Link>` wrapped with various components. Will be reverted.